.github/workflows/reusable-build-test-release.yml

```diff
@@ -1265,6 +1266,341 @@ jobs:
           path: "${{ needs.setup.outputs.directory-path }}/test-results/*.xml"
           reporter: java-junit
 
+  run-scripted-input-tests-full-matrix:
```

Comment:
Is the code the same for this and `run-scripted-input-tests-canary` job and os list the only difference? If yes, it would be better to parametrize it. I assume it is impossible to use variables for os list in matrix? To avoid code duplication we can for example:
- another action can be created 
- leave one job with all os types and just skip from inside of job body if develop

Reply:
Sure, We would take this as an enhancement in the upcoming sprint, also created a Jira for the same: https://splunk.atlassian.net/browse/AQA-522

Reply:
Ok, approved